tests/src/Api/TokenApiTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\Tests\EuropaSearchClient\Api;
+
+use GuzzleHttp\Psr7\Response;
+use OpenEuropa\EuropaSearchClient\Model\Token;
+use OpenEuropa\Tests\EuropaSearchClient\Traits\ClientTestTrait;
+use OpenEuropa\Tests\EuropaSearchClient\Traits\InspectTestRequestTrait;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * @coversDefaultClass \OpenEuropa\EuropaSearchClient\Api\TokenApi
+ */
+class TokenApiTest extends TestCase
+{
+    use ClientTestTrait;
+    use InspectTestRequestTrait;
+
+    /**
+     * @covers ::getToken
```

Comment:
Any `@covers` limits the code coverage to verify that the code in that method is covered. By running the code coverage at this moment, most of the code is reported as not covered because of the misuse of the `@covers` tags.

Reply:
done